gr-azure-software-radio/python/blob_sink.py

```diff
@@ -95,6 +87,55 @@ def __init__(self, np_dtype: np.dtype, vlen: int = 1, authentication_method: str
         self.queue = queue.Queue(maxsize=queue_size)
         self.buf = np.zeros((self.block_len*self.item_size, ), dtype=np.byte)
         self.num_buf_bytes = 0
+        self.log = gr.logger(f"gr_log.{self.symbol_name()}")
+
+        # If user accidently left .sigmf-data or meta file extension, strip it off
+        if sigmf and '.sigmf' in blob_name:
+            blob_name = blob_name.rsplit('.', 1)[0]
+
+        if sigmf:
+            meta_blob_client = self.blob_service_client.get_blob_client(container=container_name,
+                                                                        blob=blob_name + '.sigmf-meta')
+            if np_dtype == np.complex64:
+                datatype_str = 'cf32_le'
+            elif np_dtype == np.float32:
+                datatype_str = 'rf32_le'
+            elif np_dtype == np.int32: # Check that our original usage of np.int32 makes sense
```

Comment:
is there a problem with the np.int32? remove comment?

Reply:
That comment is just to refer to the one above, where I mentioned I think there is an issue with the way we are interpreting i-shorts with int32, that doesn't make sense to me